x-pack/platform/plugins/shared/stack_connectors/server/connector_types/email/send_email.ts

```diff
@@ -197,6 +208,7 @@ async function sendEmailWithNodemailer(
     subject,
     html: messageHTML,
     text: message,
+    ...(attachments && { attachments }),
   };
 
```

Comment:
do you need to include the size of the attachment in the connector usage collector information below?

Reply:
I don't think so, we pass in the entire email object that includes the attachments and we get the byteLength of that object